### DISCUSSION TITLE
A button to export a single or multiple or all frames of a sprite sheet after it has been cut into different images

### DISCUSSION THREAD

**Discussion by v4vfero** [godotengine/godot-proposals#8910] [2D]
For example, i have a 2D sprite spread sheet, and i use the built-in animation/frame cutter feature that splits the sprites into frames through the horizontal and vertical lines/grid.

Once the images/frames have been cut, it would be nice to have a button that exports these singular frames/images outside of the engine.

**Comment by Calinou** [comment]
Out of curiosity, what do you need to export individual frames for?

**Reply by v4vfero** [reply]
well, i wanted the individual frames because i wanted to edit them and share them around with my team (Friends).

We are developing a card game, and the cards are put into a big single image and we split it by 10 x 7 grid split to obtain the individual frames and each frame is the card, i wanted to export the frames into single images so i can edit the single frame and create erratas of the cards.

i realize that i can just edit the single big image or crop it myself but i feel like doing it all inside godot would be more useful.

the grid/spritesheet cropping of godot is very useful and its a great tool and i honestly prefer using it much more than gimp or photoshop grid cropping tools, godot is also way less laggy than those other tools (my potato laptop can't handle them)

i do use online tools like this (https://www.gamedeveloperstudio.com/tools/spritesheet_slicer.php) sometimes to obtain the exported single frames but i would be more "quality of life" useful if it was a feature built in godot in my opinion
